src/emm/services/modules_service.py

```diff
@@ -225,9 +232,123 @@ def git_commit_modules(self, commit: str) -> None:
 
         :param commit: Commit content for all changes.
         """
-        enabled_modules = self.get_all_modules(True)
         self.git_service.commit_all(commit)
+        enabled_modules = self.get_all_modules(True)
         for module in enabled_modules:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def validate_github_authentication(self) -> bool:
+        """Check if github CLI already authenticated."""
+        return self.github_service.validate_github_authentication()
+
+    def add_pr_links(self, comments: Dict[str, str]) -> Dict[str, List[str]]:
+        """
+        Add link to each module.
+
+        :param comments: Dictionary of module and its pull request url.
+        :return: Dictionary of pull request associate with other modules' pull requests.
+        """
+        pr_dict: Dict[str, List[str]] = defaultdict(list)
+        for module, pr_url in comments.items():
+            pr_links = "\n".join(
+                "{} pr: {}".format(repo, link) for repo, link in comments.items() if repo != module
```

Comment:
I believe github comments support markdown. It might be nice to include a little formatting to make these comments easier to read. For example using a `*` to turn the pr_links into a bulleted list.